composer.json

```diff
@@ -20,13 +20,14 @@
     ],
     "require": {
         "php": "^7.1",
-        "ezsystems/ezpublish-kernel": "^7.0@dev",
+        "ezsystems/ezpublish-kernel": "^7.5@dev",
         "ezsystems/repository-forms": "^2.0",
         "ezsystems/ezplatform-solr-search-engine": "^1.4@dev",
         "lolautruche/ez-core-extra-bundle": "^2.0",
         "twig/twig": "^1.35|^2.4.4"
     },
     "require-dev": {
+        "ezsystems/doctrine-dbal-schema": "^0.1@rc",
```

Comment:
We can remove `@rc` now, right?

Reply:
yes, I'll cleanup and rebase today or tomorrow 👍

Reply:
@andrerom Just a note. I nuked the current master and made it the dev branch for 3.4 (which will include your PR). So when rebasing, please rebase on master, not `3.3-release`.